instrumentation/action_pack/test/opentelemetry/instrumentation/action_pack/handlers/action_controller_test.rb

```diff
@@ -58,6 +58,14 @@
     _(span.attributes['code.function']).must_equal 'ok'
   end
 
+  it 'strips (:format) from http.route' do
```

Comment:
this test can probably be consolidated with another one but I see why being explicit here is helpful.